### ISSUE TITLE
Please come back to I crawler

### ISSUE THREAD

**Issue by Burgo-admin** [iCrawlerRPG/iCrawlerRPG.github.io#19]
Please come back to I crawler I miss updates and have been playing for a while

**Comment by filiperferreira** [comment]
I might...

**Comment by Burgo-admin** [comment]
🙇‍♂️

**Comment by Burgo-admin** [comment]
Magic 100 no new spell 😥

**Comment by Grimlock22180420** [comment]
I have to agree with Burgo- This game is amazing already and should 100% be continued, doesn't need to have images or anything just more things like spells, ways of leveling, more upgrades, etc. this game is one of the most fun games i've played in years(and when I say years I mean it), I would hate to see it completely abandoned when it has a lot of potential.

**Comment by Cheesy-Brik** [comment]
Agree
